scripts/update_version.py

```diff
@@ -0,0 +1,30 @@
+import fileinput
+import sys
+from pathlib import Path
+
+CURRENT_VERSION = "v0.2.1"
```

Comment:
can we pick this up somehow? otherwise it needs to be added to RELEASING.md

Reply:
Of course :) the script updates itself (`CURRENT_VERSION`) last:

`_update_version("scripts/update_version.py", new_version)`